Tools/SetupUnityPackage/UnityUtils/FileHelper.cs

```diff
@@ -19,6 +19,9 @@
 
 using System;
 using UnityEngine;
+using UnityEngine.Scripting;
+
+[assembly: AlwaysLinkAssembly]
```

Comment:
Let's put this in `AssemblyInfo.cs` - while I agree that it's a bit silly to add yet another file for only one attribute, we should follow best practices as much as possible.